test/cqlpy/conftest.py

```diff
@@ -21,10 +21,8 @@
 import time
 import random
 
-import sys
-sys.path.insert(1, sys.path[0] + '/../..')
 from test.pylib.report_plugin import ReportPlugin
-from util import unique_name, new_test_keyspace, keyspace_has_tablets, cql_session, local_process_id, is_scylla
+from test.cqlpy.util import unique_name, new_test_keyspace, keyspace_has_tablets, cql_session, local_process_id, is_scylla
```

Comment:
Again, doesn't Python support relative paths? We're already in test.cqlpy, why do we need to specify it?